frontend/server/src/main/java/org/pytorch/serve/wlm/WorkLoadManager.java

```diff
@@ -117,13 +118,14 @@ public int getNumRunningWorkers(ModelVersionName modelVersionName) {
                     // -> This may nullify process object per destroyForcibly doc.
                     if (workerProcess != null && workerProcess.isAlive()) {
                         boolean workerDestroyed = false;
-                        workerProcess.destroyForcibly();
                         try {
+                            String cmd = String.format("kill -9 %s", workerProcess.pid());
```

Comment:
* Looking at the [jdk code](https://github.com/openjdk/jdk11u-dev/blob/a09254f3d9536b9062aa9b797847667092479f18/src/java.base/share/classes/java/lang/ProcessHandleImpl.java#L348), `destroyForcibly` issue a `SIGKILL` for *nix. So, I don't think this change would make any difference.
* Also, this is *nix specific. If we are planning to support [Windows](https://github.com/openjdk/jdk11u-dev/blob/0b6306d1da0ea7023e4f6329dff8ee25337a79e6/src/java.base/windows/classes/java/lang/ProcessImpl.java#L586), this will not work.

Reply:
@maaquib :
- We are aware about the `destroyForcibly` implementation, however it does not guarantee immediate action.
- Replacing this with an independent process for sending a SIGKILL to the worker process ensures an immediate action.
- We tested the installation script on `m4.xlarge` instance, where this issue is observed consistently,  in a 10 times loop and it failed every time for `staging_0_1_1` branch and succeeded everytime with fix in this (`issue_417`) branch. 
Please refer the newly attached logs in the main PR description.
 
- I agree this is `*nix` specific fix, but works fine on Ubuntu and MacOS which we support currently. We can add a OS level check and prepare specific command whenever we want to support windows.

Reply:
> @maaquib :
> 
> * We are aware about the `destroyForcibly` implementation, however it does not guarantee immediate action.
> * Replacing this with an independent process for sending a SIGKILL to the worker process ensures an immediate action.
> * We tested the installation script on `m4.xlarge` instance, where this issue is observed consistently,  in a 10 times loop and it failed every time for `staging_0_1_1` branch and succeeded everytime with fix in this (`issue_417`) branch.
>   Please refer the newly attached logs in the main PR description.
> * I agree this is `*nix` specific fix, but works fine on Ubuntu and MacOS which we support currently. We can add a OS level check and prepare specific command whenever we want to support windows.

Tested installation around 8 times on MacOS [i.e. ~1.5GB available ram] for staging branch as well as this branch-
- **Stagging branch** -> **it failed 6 times**
- This branch -> succeeded all the times